pkg/devfile/parser/utils.go

```diff
@@ -0,0 +1,16 @@
+package parser
+
+import (
+	"fmt"
+	"path/filepath"
+)
+
+// getAbsFilePath returns the absolute representation of path for the provided file
+func getAbsFilePath(path string) (string, error) {
```

Comment:
there is a util function `GetAbsPath` already https://github.com/openshift/odo/blob/adf3006360abc5cb707d9a9f442669989272f4e5/pkg/util/util.go#L209

Reply:
Thanks for pointing this one.

Reply:
Done